rust/src/krb/log.rs

```diff
@@ -24,8 +24,12 @@ fn krb5_log_response(jsb: &mut JsonBuilder, tx: &mut KRB5Transaction) -> Result<
 {
     match tx.error_code {
         Some(c) => {
-            jsb.set_string("msg_type", "KRB_ERROR")?;
-            jsb.set_string("failed_request", &format!("{:?}", tx.msg_type))?;
+            jsb.set_string("msg_type", &format!("{:?}", tx.msg_type))?;
+            if let Some(req_type) = tx.req_type {
+                jsb.set_string("failed_request", &format!("{:?}", req_type))?;
+            } else {
+                jsb.set_string("failed_request", "UNKNOWN")?;
```

Comment:
How can we get here ?

Reply:
You arrive to this conditional when a KRB-ERROR message is logged, if the previous request type is set, then you return the request type. However, in case of the request type being None (because start capturing in the middle of an UDP connection, for example) you return the UNKNOWN value.

Reply:
Thanks. Maybe this is worth a comment in the code

Reply:
Done in https://github.com/OISF/suricata/pull/6508/commits/4642ecf72cf88f99df193b1a029f0f2ffb17850a